src/lib/components/notes/NoteExcerpt.svelte

```diff
@@ -9,7 +9,7 @@
   import { getViewerHref } from "$lib/utils/viewer";
   import { renderImage, renderPDF } from "$lib/utils/notes";
 
-  export let document = getDocument();
+  export let document: Document;
```

Comment:
In other components, we use the context value as a fallback. Can we do the same here?